WalletWasabi/Blockchain/TransactionProcessing/ProcessedResult.cs

```diff
@@ -9,12 +9,12 @@ namespace WalletWasabi.Blockchain.TransactionProcessing;
 
 public class ProcessedResult
 {
-	private Lazy<bool> _isLikelyOwnCoinJoin;
+	private Lazy<bool> _isOwnCoinJoin;
 
 	public ProcessedResult(SmartTransaction transaction)
 	{
 		Transaction = Guard.NotNull(nameof(transaction), transaction);
-		_isLikelyOwnCoinJoin = new Lazy<bool>(() => Transaction.WalletInputs.Any() && Transaction.IsLikelyCoinjoin(), true);
+		_isOwnCoinJoin = new Lazy<bool>(() => Transaction.IsOwnCoinjoin(), true);
```

Comment:
Why do we use `Lazy` here? I may be missing something but `Transaction.IsOwnCoinjoin()` looks like a cheap check. Or is for some synchronization purposes? But then `IsOwnCoinJoin` property is really used only in UI. 

So my guess would be that we don't need `Lazy<bool>` at all.